test/lint/lint-python-utf8-encoding.py

```diff
@@ -12,11 +12,10 @@
 
 from subprocess import check_output, CalledProcessError
 
-EXCLUDED_DIRS = ["src/crc32c/", "src/secp256k1/"]
```

Comment:
This is not the same list as what's in the file so you are now excluding more than before

Reply:
fixed